conctr.agent.lib.nut

```diff
@@ -195,18 +232,296 @@ class Conctr {
     }
 
 
-    //
+    // 
+    // Sets the protocal that should be used 
+    // 
+    // @param   {String}    protocol Either amqp or mqtt
+    // @return  {String}    current protocal after change
+    // 
+    function setProtocol(protocol) {
+        if (protocol == AMQP || protocol == MQTT) {
```

Comment:
If AMQP is not supported yet, the user should not be able to set it as the protocol. Maybe log an error message saying that AMQP is not supported yet if they do try to set it.